simplq/src/store/index.js

```diff
@@ -1,7 +1,8 @@
 import { combineReducers, configureStore, getDefaultMiddleware } from '@reduxjs/toolkit';
-import queuesReducer from 'store/queues';
+import queuesReducer from 'store/allQueues';
 import queueStatusReducer from 'store/queueStatus';
 import actionStatusReducer from 'store/actionStatus';
+import queueReducer from 'store/queue';
```

Comment:
`queue` reducer is also confusing since it does not follow [the naming convention](https://redux.js.org/style-guide/style-guide#name-state-slices-based-on-the-stored-data).

The slice file is named `queue.js` without a slice suffix and actually contains queue tokens. I should be probably named `queueTokensSlice.js` or `selectedQueueTokensSlice.js` with corresponding reducer and key.